src/components/shared/chart/Viewport.js

```diff
@@ -17,23 +17,48 @@ import type {
 import typeof { updateProfileSelection as UpdateProfileSelection } from '../../../actions/profile-view';
 import type { ProfileSelection } from '../../../types/actions';
 
+require('./Viewport.css');
+
 const { DOM_DELTA_PAGE, DOM_DELTA_LINE } =
   typeof window === 'object' && window.WheelEvent
     ? new WheelEvent('mouse')
     : { DOM_DELTA_LINE: 1, DOM_DELTA_PAGE: 2 };
 
-type Props = {
+type ViewportProps = {
   viewportNeedsUpdate: any,
   timeRange: StartEndRange,
   maxViewportHeight: number,
   maximumZoom: UnitIntervalOfProfileRange,
   updateProfileSelection: UpdateProfileSelection,
   selection: ProfileSelection,
-  hasZoomedViaMousewheel: () => void,
   setHasZoomedViaMousewheel: () => void,
   hasZoomedViaMousewheel: boolean,
 };
 
+type InjectedProps = {
+  containerWidth: CssPixels,
+  containerHeight: CssPixels,
+  viewportLeft: UnitIntervalOfProfileRange,
+  viewportRight: UnitIntervalOfProfileRange,
+  viewportTop: CssPixels,
+  viewportBottom: CssPixels,
+  isDragging: boolean,
+};
```

Comment:
Tell me if this needs more comments.
`ViewportProps` are the props that this component consumes, while `InjectedProps` are the props that this component injects.